public/doc-shredder/test/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
I don't think that this license is correct...

@naomiblack can you verify and prescribe the right way to do it?